src/main/scala/io/scalac/tezos/translator/service/UserService.scala

```diff
@@ -0,0 +1,47 @@
+package io.scalac.tezos.translator.service
+
+import io.scalac.tezos.translator.repository.UserRepository
+import slick.jdbc.MySQLProfile.api._
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutor, Future}
+import scala.util.Random
+import com.github.t3hnar.bcrypt._
+import io.scalac.tezos.translator.model.UserModel
+
+class UserService(repository: UserRepository, db: Database) {
+
+  private val tokenToUser = new java.util.concurrent.ConcurrentHashMap[String, String].asScala
+
+  implicit private val ec: ExecutionContextExecutor = ExecutionContext.global
+
+  private def createToken(username: String): String = {
+    def createUniqueToken = {
```

Comment:
this is minor and a matter of convention. In this project I believe the convention is keep such methods as private ones (but I might missed smth)

Reply:
This is an inner function, they're as private as it gets :)

Reply:
Yes, I understand. I am not questioning matter of scope visibility. What I mean here is that in this project afaik we do not have nested functions so far and I would prefer to keep one convention - in this case not to have nested functions defined ;)

Reply:
Ok, I get it now.  I don't think this is a convention, maybe noone needed to use nested functions yet. And if it were a convention, i would see no need to respect it, since there's nothing wrong with nested functions - IMHO visibility of any piece of code should be as limited as possible.